packages/@angular/cli/blueprints/ng/files/__path__/app/app.component.html

```diff
@@ -1,4 +1,20 @@
-<h1>
-  {{title}}
-</h1><% if (routing) { %>
+<div style="text-align:center">
+  <h1>
+    Welcome to {{title}}!!
+  </h1>
+  <img src="https://angular.io/resources/images/logos/angular/angular.png" />
```

Comment:
I agree with @hansl please just add it in-line, this way when it is removed, it's completely gone, rather than having to change it here in markup as well as removing the file from the assets dir.